Traceback (most recent call last):
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/codespace/.local/lib/python3.10/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/codespace/.local/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/local/python/3.10.13/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/codespace/.local/lib/python3.10/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/codespace/.local/lib/python3.10/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/codespace/.local/lib/python3.10/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Mencari x1, x2, x3, x4
h3 = np.array([[7, 0, -2, 0], [11.5,0.5,-3.5,-0.5], [-3,0,1,0], [-14,0,1,0]])
m = np.array([[8], [-11], [14], [3]])
print(l3@m)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[30], line 4[0m
[1;32m      2[0m h3 [38;5;241m=[39m np[38;5;241m.[39marray([[[38;5;241m7[39m, [38;5;241m0[39m, [38;5;241m-[39m[38;5;241m2[39m, [38;5;241m0[39m], [[38;5;241m11.5[39m,[38;5;241m0.5[39m,[38;5;241m-[39m[38;5;241m3.5[39m,[38;5;241m-[39m[38;5;241m0.5[39m], [[38;5;241m-[39m[38;5;241m3[39m,[38;5;241m0[39m,[38;5;241m1[39m,[38;5;241m0[39m], [[38;5;241m-[39m[38;5;241m14[39m,[38;5;241m0[39m,[38;5;241m1[39m,[38;5;241m0[39m]])
[1;32m      3[0m m [38;5;241m=[39m np[38;5;241m.[39marray([[[38;5;241m8[39m], [[38;5;241m-[39m[38;5;241m11[39m], [[38;5;241m14[39m], [[38;5;241m3[39m]])
[0;32m----> 4[0m [38;5;28mprint[39m([43ml3[49m[38;5;129m@m[39m)

[0;31mNameError[0m: name 'l3' is not defined

